src/Tools/Source/CompilerGeneratorTools/Source/VisualBasicSyntaxGenerator/GreenNodes/GreenNodeWriter.vb

```diff
@@ -848,15 +848,15 @@ Friend Class GreenNodeWriter
         ' visit all children
         For i = 0 To allChildren.Count - 1
             If allChildren(i).IsList Then
-                _writer.WriteLine("            Dim {0} = VisitList(node.{1})" + vbCrLf +
+                _writer.WriteLine("            Dim {0} = VisitList(node.{1})" + Environment.NewLine +
```

Comment:
>Environment.NewLine [](start = 80, length = 19)

It doesn't look like changes to VisualBasicSyntaxGenerator belong in this PR.

Reply:
This was from a separate PR, already merged into master. I'll rebase this out when review is complete, but it made my life simpler when running the generator.

---
In reply to: [451235615](https://github.com/dotnet/roslyn/pull/45684#discussion_r451235615) [](ancestors = 451235615)